pymc/tests/test_distributions.py

```diff
@@ -2676,6 +2676,13 @@ def ref_pdf(value):
 
                 self.check_logp(TestedInterpolated, R, {}, ref_pdf)
 
+    def test_interpolated_inits(self):
+        # Issue: https://github.com/pymc-devs/pymc/issues/5048
+        x_points = np.linspace(0, 10, 100000)
+        pdf_points = sp.norm.pdf(x_points, loc=1, scale=1)
+        with pm.Model() as m:
+            x = pm.Interpolated("interpolated", x_points, pdf_points)
```

Comment:
We should assert something, even though this was failing before the PR by itself. Perhaps eval the logp outside of the range and check it's not -inf?